aurora/__init__.py

```diff
@@ -1,15 +1,26 @@
 """Copyright (c) Microsoft Corporation. Licensed under the MIT license."""
 
 from aurora.batch import Batch, Metadata
-from aurora.model.aurora import Aurora, Aurora12h, AuroraHighRes, AuroraSmall
+from aurora.model.aurora import (
+    Aurora,
+    Aurora12hPretrained,
+    AuroraAirPollution,
+    AuroraHighRes,
+    AuroraPretrained,
+    AuroraSmall,
+    AuroraSmallPretrained,
+)
 from aurora.rollout import rollout
 from aurora.tracker import Tracker
 
 __all__ = [
     "Aurora",
-    "AuroraHighRes",
+    "AuroraPretrained",
+    "AuroraSmallPretrained",
```

Comment:
I am a little confused by what is meant by the difference between "Small" and "SmallPretrained" here